packages/E2ETest/run_wdio.js

```diff
@@ -69,26 +70,39 @@ function OverrideHyperV() {
   }
 }
 
-OverrideHyperV();
-
 const Launcher = require('@wdio/cli').default;
 
 const wdio = new Launcher('wdio.conf.js', { specs: opts });
 
 function parseLog(logfile) {
   const xmlString = fs.readFileSync(logfile);
-  let name;
+  let failures = {};
   parser.parseString(xmlString, (err, res) => {
     if (!res.testsuites) {
-      name = 'something went wrong';
+      failures = 'something went wrong';
```

Comment:
This doesn't seem right. We otherwise expect this to be an obejct instead of a string. `PrintFailedTests` seems to also expect this to be an object.